nshlib/nsh_parse.c

```diff
@@ -279,6 +279,7 @@ static const char g_arg_separator[]   = "`$";
 static const char g_redirect_out1[]   = ">";
 static const char g_redirect_out2[]  = ">>";
 static const char g_redirect_in1[]   = "<";
+static const char g_pipeline1[]       = "|";
```

Comment:
why add 1 ?
g_pipeline

Reply:
> why add 1 ? g_pipeline

May add "|&" as g_pipeline2 in the future.